samples/host_verify/host.c

```diff
@@ -161,6 +165,15 @@ int main(int argc, const char* argv[])
             "Usage:\n  %s -r <report_file>\n  %s -c <certificate_file>\n",
             argv[0],
             argv[0]);
+        fprintf(
+            stdout,
+            "Verify the integrity of enclave remote report or attestation "
```

Comment:
See earlier comment, the word "remote" here in this line looks like it may be wrong, since I don't see any discussion of how to do anything remotely.  (See also the question I posed on issue #2724)

Reply:
By "remote", this means from an enclave that is not necessarily collocated on the same CPU, e.g. described [here](https://github.com/openenclave/openenclave/blob/master/docs/DesignDocs/CustomAttestation.md) as "remote attestation".

Reply:
Correct, but this tool only accepts a local file, nothing that can be obtained from a remote enclave, hence my point about it being wrong since this tool cannot (at least using any currently  documented process) be used for doing anything about "remote" attestation.

Reply:
In SGX "remote" and "local" attestation collateral have different formats. So the report file was generated on some enclave machine in the "remote" format. Then the file made it to this machine through some means (it may have even been generated on this machine, though I know that's confusing), and then this tool can verify it. A "remote QUOTE" is basically just a "local REPORT" that is signed in a special way.

Ultimately it would be preferable that this tool can be used to verify any type of attestation collateral by loading a verifier plugin, I just think we aren't there yet in terms of what the host_verify library can do.

Reply:
> In SGX "remote" and "local" attestation collateral have different formats. So the report file was generated on some SGX machine in the "remote" format. Then the file made it to this machine through some means (it may have even been generated on this machine, though I know that's confusing), and then this tool can verify it. A "remote QUOTE" is basically just a "local REPORT" that is signed in a special way.

The above information is what I am saying is missing from any documentation.  I would expect this sample to say how to actually use it.  Can you point me to any document that says "to use this sample, run something like oecert on the remote machine, then copy the file, then run this sample"?   I can't find it, hence I believe this issue is not closed.

Reply:
Good point. I agree #2724 should be left open to capture that documentation.